### PR TITLE
HCIDOCS- 243: RN

### PR DESCRIPTION
**Pull request by apinnick**
- Adding a 4.15 RN that was overlooked.
- Requesting peer review at the same time as SME review so that this can be merged quickly.

Version(s): 4.15

Issue: [HCIDOCS-243](https://issues.redhat.com//browse/HCIDOCS-243)

Link to docs preview: [RN](https://72379--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-baremetal-raid-dell)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by GroceryBoyJr** [comment]
@apinnick hurry and grab hold of your SME, I'm doing peer review now! :-D

**Review by GroceryBoyJr — COMMENTED** [review]
@apinnick One tiny nit please - your SME should be able to confirm my comment.

**Comment by GroceryBoyJr** [comment]
@apinnick I will leave this PR assigned to me for review - I am on duty today and will help speed it along.

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -160,6 +160,12 @@ These new options include host configuration, network configuration, and baseboa
 These fields are not used during the initial provisioning of the cluster, but they eliminate the need to set the fields after installation.
 For more information, see xref:../installing/installing_with_agent_based_installer/installation-config-parameters-agent.adoc#installation-configuration-parameters-additional-bare_installation-config-parameters-agent[Additional bare metal configuration parameters for the Agent-based Installer].
 
+[id="ocp-4-15-baremetal-raid-dell"]
+==== Use the Dell iDRAC to configure a RAID during installer-provisioned installation
```
Comment (MahnoorAsghar) [inline release_notes/ocp-4-15-release-notes.adoc:164]: nit: Use the Dell iDRAC BMC (add BMC)

File: release_notes/ocp-4-15-release-notes.adoc [hunk 2]
```diff
@@ -160,6 +160,12 @@ These new options include host configuration, network configuration, and baseboa
...
+
+You can now use the Dell iDRAC with Redfish to configure a redundant array of independent disks (RAID) for the bare metal platform by using baseboard management controllers (BMCs) during an installer-provisioned installation.
```
Comment (MahnoorAsghar) [inline release_notes/ocp-4-15-release-notes.adoc:166]: You can now use the Dell iDRAC BMC with the Redfish protocol to configure a Redundant Array of Independent Disks (RAID) for the bare metal platform during an installer-provisioned installation.

Comment (apinnick) [reply release_notes/ocp-4-15-release-notes.adoc]: Thanks! I will update the PR.

**Comment by apinnick** [comment]
@GroceryBoyJr  Can you restart the 'validate-asciidoc' job? It's taking an unusually long time to finish.

**Pull request merged**